.changeset/twenty-impalas-count.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": minor
```

Comment:
```suggestion
"@comet/cms-api": major
```

Technically this is a breaking change, so it should be major.

Reply:
I was not sure, after all it's within the same mikro-orm major version.

but it doesn't make any difference as it targets next anways